setup.py

```diff
@@ -1,23 +1,34 @@
-from setuptools import setup
+#!/usr/bin/env python
+"""
+termux-create-package setup script
+https://packaging.python.org/en/latest/tutorials/packaging-projects
+"""
 
-# https://packaging.python.org/distributing/#packaging-your-project
+import setuptools
 
-def readme():
-    with open('README.rst') as f:
-        return f.read()
-setup(
-    name = 'termux-create-package',
-    version = '0.11',
-    license = 'Apache License 2.0',
-    description = 'Lightweight tool for creating deb packages',
-    long_description = readme(),
-    author = 'Fredrik Fornwall',
-    author_email = 'fredrik@fornwall.net',
-    url = 'https://github.com/termux/termux-create-package',
-    scripts = ['termux-create-package'],
-    classifiers = (
-        'Intended Audience :: Developers',
-        'Topic :: Software Development :: Build Tools',
-        'Programming Language :: Python :: 3'
-    )
+with open("README.md", "r", encoding="utf-8") as file:
+    LONG_DESCRIPTION = file.read()
+
+
+setuptools.setup(
+    name="termux-create-package",
+    version="0.11",
+    author="Fredrik Fornwall",
+    author_email="fredrik@fornwall.net",
```

Comment:
Kind of feels like a rewrite (with +5k lines) deserves a version bump and author change.. :)

Reply:
Yeah, I guess ;) I guess I can add my name to the field. Hopefully, python packaging policy hasn't changed [since 2009](https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py) to allow multiple authors.